src/modelfree/score_agent.py

```diff
@@ -63,10 +99,11 @@ def default_score_config():
         'agent_indices': None,          # which agent trajectories to save
     }
     num_env = 1                         # number of environments to run in parallel
-    episodes = 20                       # number of episodes to evaluate
-    render = True                       # display on screen (warning: slow)
+    episodes = 1                        # number of episodes to evaluate
+    render = False                      # display on screen (warning: slow)
```

Comment:
I view the common case as being interactive usage of score_agent so would vote for keeping `render=True` as default. It's easy enough to set `render = False` as being default in batch scripts etc.